Lib/Protocols/IdFTP.pas

```diff
@@ -2742,18 +2771,27 @@ procedure TIdFTP.IssueFEAT;
       Break;
     end;
   end;
-
+  if IsExtSupported('CSID') then begin {do not localize}
+      LClnt := FClientInfo.GetCSIDOutput;
+      SendCmd('CSID ' + LClnt);  {do not localize}
+      if Self.LastCmdResult.NumericCode = 200 then begin

```

Comment:
This can be simplified to `if SendCmd('CSID ' + FClientInfo.CSIDOutput') = 200 then ...`